code/data_structures/src/stack/stackUsingLinkedList/stackUsingLinkedList.cpp

```diff
@@ -0,0 +1,90 @@
+#include<iostream>
+#include<malloc.h>
```

Comment:
You shouldn't need this.

Reply:
oh means i should use (new ) keyword to allocate memory